src/components/dialogs/connectivity/connectivity-form.jsx

```diff
@@ -54,6 +55,8 @@ import {
  * @param studyUuid the study we are currently working on
  * @param currentNode the currently selected tree node
  * @param onVoltageLevelChangeCallback callback to be called when the voltage level changes
+ * @param isEquipmentModification
+ * @param previousValues
  * @returns JSX.Element
  */
 export const ConnectivityForm = ({
```

Comment:
Voltage level and busbar section fields are not editable for connectivity modification !